cluster/addons/metrics-server/resource-reader.yaml

```diff
@@ -11,6 +11,20 @@ rules:
       - nodes/metrics
     verbs:
       - get
+  - apiGroups:
+      - "apps"
+    resources:
+      - deployments
+    verbs:
```

Comment:
doesn't upstream only have get and patch?
and scoped to metrics-server only?
https://github.com/kubernetes-sigs/metrics-server/blob/1a23b5bd2e12ca55372c2cb6209c7eb1d94a2b4a/manifests/components/autoscale/rbac.yaml#L43

Reply:
> doesn't upstream only have get and patch? and scoped to metrics-server only? https://github.com/kubernetes-sigs/metrics-server/blob/1a23b5bd2e12ca55372c2cb6209c7eb1d94a2b4a/manifests/components/autoscale/rbac.yaml#L43

Thanks for the review, I have updated